extensions/interactions/TextInput/text-input-prediction.service.spec.ts

```diff
@@ -67,14 +72,16 @@ describe('Text Input Prediction Service', () => {
       const trainingData =
           window.__fixtures__['core/tests/data/text_input_training_data'];
       let correctPredictions = 0, totalAnswers = 0;
+      classifierFrozenModel = new TextClassifierFrozenModel();
+      classifierFrozenModel.model_json = JSON.stringify(classifierData);
```

Comment:
Should be modelJson if this is a frontend object.

Also can we set this using a function like TextClassifierFrozenModel.create(...)? That follows the object creation pattern that we typically use elsewhere (we don't usually go for a "create empty model and then set its properties separately" approach).

Reply:
TextClassifierFrozenModel is a typescript model compiled by proto compiler. I am afraid we can't change the API or attribute name here.

Reply:
OK sg. Please add a comment explaining that.

Reply:
Done